src/Table/Table.tsx

```diff
@@ -0,0 +1,977 @@
+            } else {
+                result = (secondItem[columnName] - firstItem[columnName]);
+            }
+        }
+        return result;
+
+    });
+    return sortedItems;
+}
+
+/**
+ * search text in array of table row
+ * @param items the array of table rows
+ * @param keyword The keyword to search in the array
+ * @param searchFields the target field to search
+ */
+function searchTextInArray(items: Array<TableRow>, keyword: string, searchFields: Array<string>): Array<TableRow> {
+    return [...items].filter((row: TableRow) => {
+        if (keyword.trim().length === 0 || searchFields.length === 0) {
+            return true;
+        }
+        return (
+            searchFields.some((searchColumn: string) => {
+                let result: boolean = false;
+                const searchField: string = searchColumn;
+                const regEx = new RegExp(keyword, "gi");
+                if (row[searchField] === null || row[searchField] === undefined) {
+                    result = false;
+                } else if (typeof row[searchField] === "string") {
+                    result = row[searchField].search(regEx) > -1;
+                } else if (typeof row[searchField] === "number") {
+                    result = String(row[searchField]).search(regEx) !== -1;
+                }
+                return result;
+            })
+        );
+    });
+}
+
+interface ActionColumnProps {
+    actionLinks?: Array<ActionLinkItem>;
+    onActionDropped?: (event: React.MouseEvent<HTMLDivElement, MouseEvent>) => void;
+    primaryActionButton?: PrimaryActionButton;
+    selectedRow: TableRow;
+    tableRef: React.RefObject<HTMLTableElement>;
+}
+
+const ActionColumn: React.FunctionComponent<ActionColumnProps> = (props: ActionColumnProps) => {
+    const btnPrimaryRandomIds = randomId("btn");
+    const [dropup, setDropup] = React.useState<boolean>(false);
+    const actionRef: React.RefObject<HTMLDivElement> = React.createRef<HTMLDivElement>();
+
+    function getActionColumnClasses() {
+        let className: string = "dropdown-content";
+        if (props.selectedRow?.actionsDropdownDropped) {
+            className += " active";
+        }
+
+        if (dropup) {
+            className += " dropup";
+        }
+
+        return className;
+    }
+
+    return (
+        <div className="action-column">
+            {props.primaryActionButton &&
+                <button
+                    id={btnPrimaryRandomIds}
+                    type="button"
+                    className="btn btn-outline-primary btn-sm"
+                    onClick={(e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+                        props.primaryActionButton.onClick(e, props.selectedRow);
+                    }}
+                >
+                    {props.primaryActionButton.label}
+                </button>
+            }
+            {props.actionLinks && props.actionLinks.length > 0 &&
```

Comment:
```suggestion
            {props.actionLinks && props.actionLinks.length &&
```

Reply:
Nope, Your condition will be true for empty array

Reply:
[].length ? true : false  returns false

Reply:
According to your code above, initializing the array with empty [] will result in truthy.  I think relying on the length is better.

Reply:
![image](https://user-images.githubusercontent.com/49055952/72698419-19ebf180-3b7f-11ea-9f96-fa0448fb5ab1.png)